guide/english/agile/chickens-versus-pigs/index.md

```diff
@@ -3,22 +3,17 @@ title: Chickens Versus Pigs
 ---
 ## Chickens Versus Pigs
 
-"Chickens versus Pigs" refers to a story about a chicken and a pig, where the chicken proposes they open a restaurant called "Ham-n-Eggs".
-The pig refuses, because while the chicken just needs to lay eggs, the pig has more at stake.
+"Chickens versus Pigs" refers to a story about a chicken and a pig, where the chicken proposes they open a restaurant called "Ham-n-Eggs". The pig refuses stating, "That arrangement doesn't work for me. While you (the chicken) appear to be willing to get involved (lay eggs), I would be fully commited".
 
-In an Agile software development project, the software developer is the pig. If you fail to complete the job, or fail to do it well,
-you have a lot at stake. This could be your reputation, or maybe even your position. Other team members might also be considered pigs, 
-depending on how much they have at stake. 
+In an Agile software development project, the software developer is represented by the pig. If you fail to complete the job, or fail to do it well, you have a lot at stake. This could be your reputation, or maybe even your position. Other team members might also be considered pigs, depending on how much they have at stake. 
 
-On the other hand, many team members are chickens. For example, the client or a high-level project manager would not really be impacted
-by the failure of the project. They are interested in its success, and might make contributions, but have lower stakes and thus
-have significantly less commitment to the project. 
+On the other hand, many team members are chickens. For example, the client, an executive or a project manager would not really be impacted by the failure of the project. They are interested in its success, and might make contributions, but have less at risk and thus are significantly less commitmented to the project. 
```

Comment:
A grammatical change here,
the word `commitmented` isn't correct.
Instead perhaps use the word `committed` here ?